.buildkite/scripts/resolve_es_branch.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+VERSION_URL="https://raw.githubusercontent.com/elastic/logstash/main/ci/logstash_releases.json"
+
+echo "Fetching versions from $VERSION_URL"
+VERSIONS=$(curl --silent $VERSION_URL)
+
+if [[ "$SNAPSHOT" == "true" ]]; then
+  key=$(echo "$VERSIONS" | jq '.snapshots."'"$ELASTIC_STACK_VERSION"'"')
```

Comment:
if you use `-r` with `jq` like: `jq -r '.snapshots."'"$ELASTIC_STACK_VERSION"'"')` you won't need to remove wrapped quotation marks later:

```
$ key=$(echo $VERSION_URL | jq '.snapshots."'"8.x"'"')
$ echo $key
"8.12.0-SNAPSHOT"
```

versus

```
$ key=$(echo $VERSION_URL | jq -r '.snapshots."'"8.x"'"')
$ echo $key
8.12.0-SNAPSHOT
```